kolibri/core/discovery/tasks.py

```diff
@@ -380,3 +412,11 @@ def reset_connection_states(broadcast_id):
             job_id=generate_job_id(TYPE_CONNECT, static_location_id),
             args=(static_location_id,),
         )
+
+    # For KDP, set the application to 'Kolibri Data Portal' without enqueuing update task
+    kdp_location = NetworkLocation.objects.filter(
+        id=DATA_PORTAL_BASE_INSTANCE_ID
+    ).first()
+    if kdp_location:
+        kdp_location.application = "Kolibri Data Portal"
```

Comment:
Non-blocker, but it looks to me that the applications is already being set [here](https://github.com/learningequality/kolibri/pull/12703/files#diff-f96ed0378ddc60c498127f1188aa07a8b77d78ea3d898c41fa72dc28df45f812R375). Is there a particular reason why it's being updated here? Its probably better to save this as constant for better `management` of the application name?

Reply:
No, there's no reason for this to be set twice, thanks for catching this!